myapp/app/controllers/tasks_controller.rb

```diff
@@ -56,6 +69,15 @@ def edit
     @task = Task.find_by(id: params[:id])
     return redirect_to error_path(404) if @task.nil?
     return redirect_to error_path(401) if @task.user_id != current_user.id
+
+    @labels_value = ''
+    labels = Label.joins(:tasks_labels).where('tasks_labels.task_id': params[:id])
+    if labels.any?
+      labels.each do |label|
+        @labels_value += ' ' unless @labels_value.empty?
+        @labels_value += label.name
+      end
+    end
```

Comment:
[FYI] You can write this as a one-liner.
```suggestion
        @labels_value = labels.map(&:name).join(' ') if labels.any?
```

Reply:
Nice ruby trick! Committed the suggestion.